scripts/upgrade/upgrade_bridge.py

```diff
@@ -97,13 +97,19 @@ def main():
     bridge = connect_bridge(badger, badger_config.prod_json)
     swap = connect_swap(badger_config.prod_json)
 
-    upgrade_bridge(badger, bridge)
-    console.print("[orange]Queued bridge adapter update[/orange]")
+    # new proxy deployed at 
+    # https://etherscan.io/address/0xd120b381296aa255e4acd9dc80e032df2223fc6a#writeProxyContract
+    # upgrade_bridge(badger, bridge)
```

Comment:
these are some of the first steps as per henry's instructions. they were done in a separate deployment before looking here. 

@notbdu can you confirm that we can skip these ones i commented out? 

my thinking is i only need to do two more things for the new deployed contracts (unless these old ones are correct) 
- `changeProxyAdmin` to `devProxyAdmin` on current proxy
- `transferOwnership` to `devMultisig` on badger bridge adapter implementation

then someone will need to execute this script to queue the txs (myself i can do it i think?)

Reply:
Does the latest OZ proxy contract work with badger's dev proxy admin contract? Might be worth testing that bit in isolation as I'm not sure if they're backwards compatible across versions.

I think if we're deploying a brand new set of contracts + proxy we can remove the upgrade code.

Not sure how much is left to configure in the script below.

We'd need to update all the pointers in the config json to the newly deployed contracts and make sure the tests still run. If there are deploy things to be done, should probably do the following in fork mode:
- run this upgrade script
- run all tests while connected to new contract addrs

If tests pass, the script/deploy _should_ be correct.